forui/lib/src/widgets/popover.dart

```diff
@@ -0,0 +1,283 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:meta/meta.dart';
+
+import 'package:forui/forui.dart';
+
+/// A controller that controls whether a [FPopover] is shown or hidden.
+final class FPopoverController extends ChangeNotifier {
```

Comment:
Might be missing something here, but do we need `ChangeNotifier`? `notifyListeners()` doesn't seem to be called anywhere.